patches/v8/0004-Allow-Windows-builds-under-Bazel.patch

```diff
@@ -1,46 +1,46 @@
-From 7958c73efccb179171fcf9072a179f6b1aa67e24 Mon Sep 17 00:00:00 2001
+From 10c8b0b828769d8a0dd3cf454704785a270fb2a6 Mon Sep 17 00:00:00 2001
 From: Brendan Coll <bcoll@cloudflare.com>
 Date: Thu, 16 Mar 2023 11:56:10 +0000
 Subject: Allow Windows builds under Bazel
 
 
 diff --git a/BUILD.bazel b/BUILD.bazel
-index 1f269f9c149e56a7098697fc2346b5998b0c6e01..89bc2f54fc5e42a2cf6d6fa41742c2dfe2c9ab48 100644
+index f2b2f4da0f8164ef29e36a3d1e9489b8e9c7b86e..84a281458aaa18525fa3640b3710414153e008b1 100644
 --- a/BUILD.bazel
 +++ b/BUILD.bazel
-@@ -817,6 +817,7 @@ filegroup(
+@@ -820,6 +820,7 @@ filegroup(
          "src/base/platform/mutex.h",
          "src/base/platform/platform.cc",
          "src/base/platform/platform.h",
 +        "src/base/platform/platform-posix.h",  # Always included by src/execution/isolate.h
```

Comment:
If I'm not mistaken, it looks like this change was introduced in the 13.0 update. Same with the removed line below. So this doesn't look like a reverted file. Could you cross-check? Maybe I'm not reading this correctly.

Reply:
It's just a revert, you can look at the PR commit by commit to verify, the first commit is the revert commit

Reply:
Dan points out that this is github miscolouring the patch file.  This line was in the patch file all the time. Does that answer your question or is it still suspicious?